tests/Unit/User/UserTest.php

```diff
@@ -36,7 +36,7 @@ public function testReturnsView(): void
     public function testWithActivities(): void
     {
         $this->assertEquals(
-            ActivityCollection::make($this->user->activities)->toArray(request()),
+            ActivityCollection::make($this->user->actions)->toArray(request()),
```

Comment:
This test does not make sense. The controller isn't doing anything. The User is only for testing, so that doesn't need a test. This needs another look.